docs/en/operations/table_engines/stripelog.md

```diff
@@ -0,0 +1,86 @@
+# StripeLog
+
+Engine belongs to the [family of log engines](log_family.md).
+
+Use this engine in scenarios, when you need to write many tables with the small amount of data (less than 1 million rows).
```

Comment:
probably should add that you also shouldn't need efficient access to subset of columns, otherwise use Log

Reply:
It looks like this difference described enough in the common article.